src/components/TodoForm.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import usersFromServer from '../api/users';
+
+interface TodoFormProps {
+  onAddTodo: (title: string, userId: number) => void;
+}
+
+export const TodoForm: React.FC<TodoFormProps> = ({ onAddTodo }) => {
+  const [userSelectId, setUserSelectId] = useState(0);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [errorUserSelectId, setErrorUserSelectId] = useState(false);
+  const [errorTodoTitle, setErrorTodoTitle] = useState(false);
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    if (userSelectId === 0) {
+      setErrorUserSelectId(true);
+    }
+
+    if (todoTitle === '') {
+      setErrorTodoTitle(true);
+    }
+
+    if (userSelectId === 0 || todoTitle === '') {
+      return;
+    }
+
+    onAddTodo(todoTitle, userSelectId);
+    setUserSelectId(0);
+    setTodoTitle('');
+  };
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
[CODE STYLE] - Avoid using one-letter variable names. It's better to use descriptive names for your variables. For example, instead of `e`, use `event` or `changeEvent`. This makes your code easier to read and understand.